### ISSUE TITLE
Perspectives of Visually Impaired Visitors on Museums: Towards an Integrative and Multisensory Framework to Enhance the Museum Experience

### ISSUE THREAD

**Issue by Wotipati** [Wotipati/Blind_Accessibility_Papers#436]
DSAI 2020: 9th International Conference on Software Development and Technologies for Enhancing Accessibility and Fighting Info-exclusion

### Links
- Paper: https://dl.acm.org/doi/10.1145/3439231.3439272

### Abstract
Although the growing concern to make museums accessible to individuals with visual impairments, their participation in these institutions is still limited, given the multiple barriers they often experience and the lack of assistive technologies to promote access. This research investigates the perspectives of 72 blind and partially sighted individuals on enhancing their visiting experience in museums. A co-created framework to improve visitors’ autonomy is proposed, concluding that sensory, intellectual and physical access must be integrated into the pre, on-site and post phases of visiting museums.

- 72名の視覚障害者にインタビューしmuseumを訪れる体験を向上させるための改善策を調査
